resources/js/src/app/main.js

```diff
@@ -121,19 +121,22 @@ var init = (function($, window, document)
             var offset = 250;
             var duration = 300;
 
-            $(window).scroll(function()
+            if (window.matchMedia("(min-width: 768px)").matches)
```

Comment:
Vorsicht: Die Bedingung wird bei initialen rendern der Seite ausgewertet. D.h. dass beim Resize wird der Listener auf das Scroll-Event nicht mehr registriert.
Ich würde die Abfrage daher in die Callback-Funktion des Listeners schreiben.